tools/build_versions.py

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env python
 
-import numpy as np
-import scipy as sp
-import matplotlib as mpl
-from PIL import Image
-import Cython
-import networkx
+from pathlib import Path
+import requirements
+import importlib
+import pkg_resources
 
+script_dir = Path(__file__).parent / '..' / 'requirements'
 
-for m in (np, sp, mpl, Image, networkx, Cython):
-    if m is Image:
-        version = m.VERSION
-    else:
-        version = m.__version__
-    print(m.__name__.rjust(10), ' ', version)
+for requirement_file in script_dir.glob('*.txt'):
+    if requirement_file.name == 'numpy.txt':
+        continue
+    print(requirement_file.name)
+    with open(str(requirement_file), 'r') as fd:
+        for req in requirements.parse(fd):
+            try:
+                version = pkg_resources.get_distribution(req.name).version
+                print(req.name.rjust(20), version)
```

Comment:
Why do we need `rjust` here?

Reply:
Looks pretty? Notice how in the same output, all the version numbers and package names are aligned.

Reply:
Yes, sorry. I've misread it as `rsplit` :sweat_smile: .